test/test_gaussian.py

```diff
@@ -342,3 +343,15 @@ def test_integrate_gaussian(int_inputs, real_inputs):
     exact = Integrate(log_measure, integrand, reduced_vars)
     assert isinstance(exact, Tensor)
     assert_close(approx, exact, atol=0.1, rtol=0.1)
+
+
+@pytest.mark.xfail(reason="numerically unstable")
+def test_mc_plate_gaussian():
+    log_measure = Gaussian(torch.tensor([0.]), torch.tensor([[1.]]),
+                           (('loc', reals()),)) + torch.tensor(-0.9189)
+    integrand = Gaussian(torch.randn((100, 1)) + 3., torch.ones((100, 1, 1)),
+                         (('data', bint(100)), ('loc', reals())))
+    with interpretation(monte_carlo):
+        res = Integrate(log_measure, integrand, frozenset({'loc'}))
+        res = res.reduce(ops.mul, frozenset({'data'}))
+        assert not torch.isinf(res).any()
```

Comment:
@fritzo the minipyro example in this PR now fails with an `inf` loss.  This test demonstrates the fragment of computation within the ELBO that is the source of the numerical instability.